Lzw/Lzw/LzwMethods.cs

```diff
@@ -0,0 +1,74 @@
+﻿namespace Lzw;
+
+public static class LzwMethods
+{
+    /// <summary>
+    /// Compresses file using Lzw algorithm
+    /// </summary>
+    /// <param name="path">Path to file</param>
+    public static void Compress(string path) 
+    {
+        var compressedPath = path + ".zipped";
+        using var fileRead = File.OpenRead(path);
+        using var fileWrite = File.OpenWrite(compressedPath);
+        var trie = new TrieForLzw();
+        for (var charIndex = 0; charIndex < fileRead.Length; charIndex++)
+        {
+            var valueByte = (byte) fileRead.ReadByte();
```

Comment:
```suggestion
            var valueByte = (byte)fileRead.ReadByte();
```
Приведение типов пробелом не отделяется.

Reply:
Не исправляем проблему, ставим resolved, профит. Вообще, как говорил один мой знакомый, "Resolved — для учителя!" :)